lib/fca_api/response.rb

```diff
@@ -0,0 +1,15 @@
+module FcaApi
+  class Response
+    SUCCESS_MESSAGE = 'ok'
+      
+    attr_reader :response
+
+    def initialize(response)
+      @response = response
+    end
+
+    def ok?
+      response.body['Message'].downcase.include?(SUCCESS_MESSAGE)
```

Comment:
just to double check.... Do API responses always include a body with a 'Message'? Trying to avoid situations where the API does not return what we expect and instead of a controlled `ok? false` response it blows up :sweat_smile:

Reply:
For the firm endpoint yes. I was trying to not pre-empt future uses as it's more likely that with additional endpoints we'll have a better idea of how to structure the Response class.